miso-web/src/main/webapp/WEB-INF/pages/editSample.jsp

```diff
@@ -228,7 +228,7 @@
     </tr>
     <c:choose>
     <c:when test="${not detailedSample}">
-	  <tr style="background-color:yellow;">
+	  <tr class="highlightfield">
```

Comment:
https://github.com/TGAC/miso-lims/pull/1881 removes the two changed elements on this page - the synthetic box and the QC status stripe, so these changes will likely be overwritten

Reply:
OK.